aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -187,13 +187,22 @@ pub enum EntryFunctionCall {
     },
 
     /// Creates a new resource account, publishes the package under this account transaction under
-    /// this account and leaves the signer cap readily available for pickup.
+    /// this account, and leaves the signer cap readily available for pickup.
     ResourceAccountCreateResourceAccountAndPublishPackage {
         seed: Vec<u8>,
         metadata_serialized: Vec<u8>,
         code: Vec<Vec<u8>>,
     },
 
+    /// Creates a new resource account, publishes the package under this account transaction under
+    /// this account with metadata, and leaves the signer cap readily available for pickup.
+    ResourceAccountCreateResourceAccountAndPublishPackageWithData {
```

Comment:
OMG

Reply:
:smiling_imp:

Reply:
🤦 Can we fix the comments here too?

And, in earnest, is there not a better name for this?